storage-manager/src/IOCoordinator.cpp

```diff
@@ -1137,7 +1187,10 @@ boost::shared_array<uint8_t> IOCoordinator::mergeJournal(const char *object, con
     
```

Comment:
The fixes to mergeJournal and related functions were all around little KPI accounting flaws & error propagation.  Should not affect functionality.